userguide/snippets/pathlengths.rst

```diff
@@ -34,11 +34,27 @@ can also reduce the length of these limits by one or more bytes.
    | Names               |                |                                                                        |
    +---------------------+----------------+------------------------------------------------------------------------+
    | Mounted Filesystem  | 88 bytes       | Mounted filesystem path length (*MNAMELEN*). Longer paths can prevent  |
-   | Paths               |                | a device from being mounted.                                           |
+   | Paths               |                | a device from being mounted.  See below for examples of path length    |
+   |                     |                | calculation.                                                           |
    +---------------------+----------------+------------------------------------------------------------------------+
    | Device Filesystem   | 63 bytes       | `devfs(8)                                                              |
    | Paths               |                | <https://www.freebsd.org/cgi/man.cgi?query=devfs&sektion=8>`__ device  |
    |                     |                | path lengths (*SPECNAMELEN*). Longer paths can prevent a device from   |
    |                     |                | being created.                                                         |
    +---------------------+----------------+------------------------------------------------------------------------+
 
+Care is needed with regard to ZFS snapshots. **If the mounted path length for a snapshot exceeds 88 characters, 
+then the snapshot and its data will be safe but inaccessible until its mounted path length is reduced**. 
+Typically this is rectified by renaming (shortening) the dataset, snapshot, or mountpoint, so that the path length is
+below the 88 byte limit. Note that ZFS snapshots are automatically mounted on demand within hidden :file:`/.zfs/snapshot/`
```

Comment:
The meaning of "on demand" is not clear here.

Reply:
reworded

Reply:
The "88 byte limit" sentence is addressed in the previous comment.
I agree with Warren, is "on demand" needed here? This sentence starts with Note. Generally a bad idea. Can this be turned into an actual note admonition? See next comment.

Reply:
Re-reading this along with the next section, it sounds like snapshots are dynamically mounted when the data in the snapshot is needed. This could be from entering that dir via a GUI file browser, or via the CLI. If this is the case, I'd say expand on this a bit to describe that.

Reply:
@miker - The text was reworked some time ago after Warren's comment, the expressions "Note:" and "on demand" aren't in the current version.